android/app/build.gradle

```diff
@@ -153,8 +153,9 @@ android {
 
 dependencies {
     implementation 'androidx.work:work-runtime:2.7.1'
+    implementation 'androidx.appcompat:appcompat:1.6.1'
     androidTestImplementation('com.wix:detox:+')
-    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.appcompat:appcompat:1.7.0'
```

Comment:
androidx.appcompat:appcompat is added twice with different version numbers, one should be enough?

Reply:
Fixed